### DISCUSSION TITLE
Turbsim API wind

### DISCUSSION THREAD

**Discussion by ghost** [OpenFAST/openfast#1073] [Q&A]
I am using Turbsim 2.0 and trying to use the API turbulent model and wind profile model. 

I noticed that when we use xEWM50 (extreme 50-year wind model) for the IEC_wind type, we cannot use URef (mean velocity at the reference height). xEWM50 and NTM have different wind shear so I am wondering if we use the NTM to generate any 50-year extreme wind for a user-defined target wind speed. Is it a typical practice to use NTM with URef for generating extreme wind?

**Comment by jjonkman** [comment]
Dear @jinch999,

I'm not sure I understand your question, but does the TurbSim User's Guide v2.0 answer your question: https://www.nrel.gov/wind/nwtc/assets/downloads/TurbSim/TurbSim_v2.00.pdf?

Best regards

**Reply by ghost** [reply]
I read that manual before; but, I still do not get any information. 
If we have 50-year wind information on a certain site, can we just select the NTM with the API model to represent any extreme wind (since 50-yr is regarded as extreme wind)?

**Comment by bjonkman** [comment]
I'm not sure the EWM models should be used with the API models. In fact--as mentioned in the TurbSim manual--`IEC_WindType` is only used with **IEC** spectral models (not API).  Also note that the API model is based on 1-hr mean values at 10 m above sea level, while the EWM models assume 10-minute mean values at hub height. 

With the `xEWM1` and `xEWM50` models, the value of `VRef` is specified in the IEC standard and is based on the wind turbine class (the value of _x_), not user input (that is, not `uRef` and not `vHub`). If you want to use an extreme wind profile, you can use the a power law profile and specify any values for shear and hub-height velocity you wish. The equations used in each of the models are in the TurbSim manual, so you can adjust the inputs to get the output you desire. But keep in mind, not all inputs from the TurbSim input file apply if you are using the API wind profile.
